shishir/Day1/activity_main.xml

```diff
@@ -0,0 +1,168 @@
+    tools:context="com.example.myapplicationapplication.MainActivity"
+    tools:layout_editor_absoluteY="81dp"
+    tools:layout_editor_absoluteX="0dp">
+
+
+
+    <ImageView
+        android:src="@drawable/images"
+        android:layout_width="350dp"
+        android:layout_height="200dp"
+        android:scaleType="center"
+        android:id="@+id/imageView"
+        android:layout_marginLeft="8dp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_marginRight="8dp"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintHorizontal_bias="0.488"
+        android:layout_marginStart="8dp"
+        android:layout_marginEnd="8dp" />
+    <EditText
+        android:id="@+id/editText"
+        android:layout_width="367dp"
+        android:layout_height="50dp"
+        android:background="@android:color/white"
+        android:layout_marginRight="1dp"
+        android:ems="10"
+        android:inputType="textPersonName"
+        android:paddingLeft="10dp"
+        android:textColor="@android:color/black"
+        android:text="jacobmoore001@gmail.com"
+        app:layout_constraintRight_toRightOf="@+id/editText2"
+        android:layout_marginTop="8dp"
+        app:layout_constraintTop_toBottomOf="@+id/imageView"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintVertical_bias="0.059"
+        android:layout_marginEnd="1dp" />
+
+
+    <EditText
+        android:id="@+id/editText2"
+        android:layout_width="369dp"
+        android:layout_height="52dp"
+        android:paddingLeft="10dp"
+        android:background="@android:color/white"
+        android:layout_marginLeft="20dp"
+        android:layout_marginRight="20dp"
+        android:ems="10"
+        android:inputType="textPassword"
+        android:textColor="@android:color/black"
+        android:text="1234567"
+        app:layout_constraintRight_toRightOf="@+id/imageView"
+        app:layout_constraintLeft_toLeftOf="@+id/imageView"
+        android:layout_marginTop="8dp"
+        app:layout_constraintTop_toBottomOf="@+id/editText"
+        android:layout_marginBottom="8dp"
+        app:layout_constraintBottom_toTopOf="@+id/textView2"
+        app:layout_constraintVertical_bias="0.0"
+        app:layout_constraintHorizontal_bias="0.491"
+        android:layout_marginStart="20dp"
+        android:layout_marginEnd="20dp" />
+
+    <Button
+        android:id="@+id/button"
+        android:layout_width="367dp"
+        android:layout_height="48dp"
+        android:background="#293e6a"
+        android:text="Log In"
+        android:textColor="@android:color/white"
+        app:layout_constraintRight_toRightOf="@+id/editText2"
+        app:layout_constraintLeft_toLeftOf="@+id/editText2"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
again bottom constraint should be changed. you must guess it by now, every bottom should be constrained to top of the element below